### PR TITLE
[OMA-949] - Add MoPub version and error message

### PR DESCRIPTION
**Pull request by PabloGarciaAldehuela**

### CODE REVIEW

File: app/src/main/res/layout/fragment_nav_ads.xml [hunk 1]
```diff
@@ -115,6 +115,18 @@
             android:text="@string/action_show"
             android:visibility="gone" />
 
+        <TextView
```
Comment (eros902002) [inline app/src/main/res/layout/fragment_nav_ads.xml:118]: For this error message I'd add a small label like the other components have that states this is the error message. Similar to HyBid demo app. Also I'd add the same margins the other components have.

Comment (PabloGarciaAldehuela) [reply app/src/main/res/layout/fragment_nav_ads.xml:118]: Added label

Comment (PabloGarciaAldehuela) [reply app/src/main/res/layout/fragment_nav_ads.xml:118]: Done

**Review by eros902002 — APPROVED** [review]
Looks good : )

**Pull request merged**